### PR TITLE
Fix: Marquez web displays invalid time format in job and dataset deta…

### PR DESCRIPTION
**Pull request by kachontep**
This is a pull request for fixing the issue #1306

### CODE REVIEW

File: web/src/__tests__/helpers/index.test.ts [hunk 1]
```diff
@@ -3,11 +3,17 @@ const datasets = require('../../../docker/db/data/datasets.json')
...
 describe('formatUpdated Function', () => {
-  const updatedAt = ''
-  const formatedDate = formatUpdatedAt(updatedAt)
   it('Should return an empty string when passed a falsey value', () => {
+    const updatedAt = ''
+    const formatedDate = formatUpdatedAt(updatedAt)
     expect(formatedDate).toBe('')
   })
+  it('Should return a datetime string in format like "May 1, 2021 01:45pm"', () => {
+    // const updatedAt = '2021-05-13T13:45:13.456+07:00'
```
Comment (phixMe) [inline web/src/__tests__/helpers/index.test.ts:12]: Let's remove this comment here.

File: web/globalSetup.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = async () => {
+  process.env.TZ = 'UTC'
+}
```
Comment (phixMe) [inline web/globalSetup.ts:3]: Can you just add a newline here?

**Review by phixMe — COMMENTED** [review]
Mostly looks good, just a few minor things.

**Review by wslulciuc — APPROVED** [review]
Thanks for the fix, @kachontep! 👍

**Pull request merged**